data/publiccloud/azure_lb.sh

```diff
@@ -0,0 +1,296 @@
+
+## Show vnet using first item returned from net list
+echo "Network vnet list & show"
+echo "***************************"
+vnlist=$( az network vnet list -g ${grpname} --query "[].{name:name}" -o tsv | head -n 1)
+cmd_status "az_net_vnet_show" az network vnet show -n ${vnlist} -g ${grpname} -o table
+
+# Create a public IP address for load balancer.
+echo "Creating $loadBalancerPublicIp"
+cmd_status "az_net_pubip_create" az network public-ip create --resource-group ${grpname} --name ${loadBalancerPublicIp}
+
+## Show public-ip using first item returned from public-ip list
+echo "Network public-ip list & show"
+echo "***************************"
+iplist=$(az network public-ip list -g ${grpname} --query "[].{name:name}" -o tsv | head -n 1)
+cmd_status "az_net_pubip_show" az network public-ip show -n ${iplist} -g ${grpname} -o table
+
+
+# Create an Azure Load Balancer.
+echo "Creating ${loadBalancer} with ${frontEndIp} and ${backEndPool}"
+cmd_status "az_net_lb_create" az network lb create \
+    --resource-group "${grpname}" \
+    --name "${loadBalancer}" \
+    --public-ip-address "${loadBalancerPublicIp}" \
+    --frontend-ip-name "${frontEndIp}" \
+    --backend-pool-name "${backEndPool}" \
+    --output table
+
+echo "Load Balancer list & show"
+echo "***************************"
+cmd_status "az_net_lb_show" az network lb show --name "${loadBalancer}" --resource-group "${grpname}" -o table
+
+# Create an LB probe on port 80.
+echo "Creating ${probe80} in ${loadBalancer}"
+cmd_status "az_net_lb_pr_create" az network lb probe create \
+   --resource-group "${grpname}" \
+   --lb-name "${loadBalancer}" \
+   --name "${probe80}" \
+   --protocol tcp \
+   --port 80
+
+echo "LB probe list & show"
+echo "***************************"
+cmd_status "az_net_lb_pr_show" az network lb probe show \
+   --name "${probe80}" \
+   --resource-group "${grpname}"  \
+   --lb-name "${loadBalancer}" \
+   --output table
+
+
+# Create an LB rule for port 80.
+echo "Creating ${loadBalancerRuleWeb} for ${loadBalancer}"
+cmd_status "az_net_lb_rule_create" az network lb rule create \
+   --resource-group "${grpname}"  \
+   --lb-name "${loadBalancer}" \
+   --name "${loadBalancerRuleWeb}" \
+   --protocol tcp \
+   --frontend-port 80 \
+   --backend-port 80 \
+   --frontend-ip-name "${frontEndIp}" \
+   --backend-pool-name "${backEndPool}" \
+   --probe-name "${probe80}" \
+   --output table
+
+echo "LB rule list & show"
+echo "***************************"
+iplist=$(az network lb rule list -g ${grpname} --lb-name ${loadBalancer} --query "[].{name:name}" -o tsv | head -n 1)
+cmd_status "az_net_lb_rule_show" az network lb rule show \
+   --name "${iplist}" \
+   --resource-group "${grpname}" \
+   --lb-name "${loadBalancer}" \
+   --output table
+
+# Create three NAT rules for port 22.
+echo "Creating three NAT rules named ${loadBalancerRuleSSH}1 - ${loadBalancerRuleSSH}3"
+for i in `seq 1 3`; do
```

Comment:
```suggestion
for i in {1..3}; do
```
No need to call an external command.  I think `for i in 1 2 3` is even more readable.

Reply:
done